internal/quaiapi/quai_api.go

```diff
@@ -794,8 +807,15 @@ func (s *PublicBlockChainQuaiAPI) GetPendingHeader(ctx context.Context) (map[str
 	// Only keep the Header in the body
 	pendingHeaderForMining := pendingHeader.WithBody(pendingHeader.Header(), nil, nil, nil, nil, nil)
 	// Marshal the response.
-	marshaledPh := pendingHeaderForMining.RPCMarshalWorkObject()
-	return marshaledPh, nil
+	protoWo, err := pendingHeaderForMining.ProtoEncode(types.PEtxObject)
```

Comment:
I'm curious why you are encoding it as a PEtxObject view instead of PhObject but this is probably just a misunderstanding of the view types on my part

Reply:
PETx object just has the workobject header and the Header in hte body which is all we need, thats all, i can change the name if you think its confusing.